webflow-hacks/misc-hacks/12.ts

```diff
@@ -0,0 +1,15 @@
+// JS
+// on button click
+document.querySelector<HTMLElement>('.hack-button.hack12-js')?.addEventListener('click', ()=>{	
+  // paste your function - start
+  (document.querySelector('.hack12-text.hack12-js') as HTMLElement).innerHTML = 'This is me on Webflow.';
+  // paste your function - end
+});
+
+// jQuery
+// on button click
+document.querySelector<HTMLElement>('.hack-button.hack12-jquery')?.addEventListener('click', ()=>{	
+  // paste your function - start
+  (document.querySelector('.hack12-text.hack12-jquery') as HTMLElement).innerHTML = 'This is me on Webflow.';
+  // paste your function - end
+});
```

Comment:
Same here.
- Don't use type assertions.
- Perform null checks.
- Run Prettier.
- Don't chain query selectors with other logic. Store the DOM elements in variables, it's much cleaner.

Reply:
![Screenshot_4](https://user-images.githubusercontent.com/90381638/151357225-d4f68807-d291-4b9c-989e-e944fba741ec.png)

I think there is something wrong with my prettier. I've been having same error when i try to write the code without assertion.

Reply:
Have you run `npm install`?

Reply:
yes i have, several times.

Reply:
![Screenshot_6](https://user-images.githubusercontent.com/90381638/151358353-188ed89b-c223-4148-8204-8d9a6e450bd1.png)
Here

Reply:
What OS are you using? Linux?

Reply:
Windows 10

Reply:
I think there is some issue with the configuration. I noticed something, if i do this: `if (element) {elemet.textContent}` it works because it under condition. but out side condition it doesn't work.

Reply:
Any solution regarding this